### ISSUE TITLE
[A11y] Backend: Manage Leases

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3358]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Backend

### Admin user?

Yes

### Issue Description

Reference Link: https://oregondigital.org/leases?locale=en

Logged in sidebar Tasks > Manage Leases

Recieved the following errors on Accessibility Insights

<img width="941" alt="Image" src="https://github.com/user-attachments/assets/5f22ed60-a370-49bb-899f-e75969065894" />

### Suggested Fix

Fix any of the following:

Element has insufficient color contrast of 2.48 (foreground color: #ffffff, background color: #5cb85c, font size: 9.0pt (12px), font weight: bold). Expected contrast ratio of 4.5:1

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Revalidate the page using Accessibility Insights
